docs/user-docs/latest/plugins/strapi-plugins.md

```diff
@@ -0,0 +1,32 @@
+---
+title: List of Strapi plugins - Strapi User Guide
+description: Reference guide to Strapi plugins explaining how they work and how they expand a Strapi application
+---
+
+# List of Strapi plugins
+
+::: warning 🚧 This section of the user guide is a work in progress. Stay tuned!
+<br>
+:::
+
+No matter if they are installed by default, or additional, Strapi plugins allow to expand your application by adding more options and possibilities. All options are documented in their right places throughout the user guide. However, you can use the following documentation as a reference guide to know which Strapi plugins are available, how they work and which options they add to your Strapi application.
+
+## Additional plugins
+
+### <img width="28" src="../assets/plugins/icon_i18n-plugin.png"> Internationalization plugin
```

Comment:
Should we add the plugin emoji there?
If yes, should we do the same thing with the dev docs? @pwizla

Reply:
Yes I added it but as an image, because for some reason when I added the emoji directly, the link to the related title didn't work properly 🤔  But also, since not all plugins have an emoji (some have non-emoji images to represent them) I thought that for consistency (in the TOC for instance) it would be better to embed all of them as images.
And yes we are doing the same for the dev docs!